generated/nifake_non_ivi/nifake_non_ivi_service.cpp

```diff
@@ -800,3 +802,8 @@ namespace nifake_non_ivi_grpc {
   }
 } // namespace nifake_non_ivi_grpc
 
+namespace nidevice_grpc {
```

Comment:
TODO: DAQ and fake-non-ivi are defeating my conditional around this. I think because they have custom types that don't codegen the converters (timestamps)